src/buildfromsource/FSharp.Compiler.Private/FSComp.resx

```diff
@@ -4343,6 +4355,6 @@
     <value>A type annotated with IsReadOnly must also be a struct. Consider adding the [&lt;Struct&gt;] attribute to the type.</value>
   </data>
   <data name="chkStructsMayNotReturnAddressesOfContents" xml:space="preserve">
-    <value>Struct members cannot return 'this' or fields by reference</value>
+    <value>Struct members cannot return the address of fields of the struct by reference</value>
```

Comment:
not sure why this line is changed, might be due to rebase or inconsistent file (we shouldn't checkin generated files).

Reply:
Don't worry about it, some file didn't get updated when it should have been, this is now updating it

Reply:
Ok, but then can we make a plan to remove the files that are generated after an initial `build` from the repository? 

I think it would help to know what is canonical file, remove burden in code reviews and improve other maintainability aspects.